include/SFML/Window/Event.hpp

```diff
@@ -41,6 +41,8 @@
 
 namespace sf
 {
+class WindowBase;
+
```

Comment:
You don't need this. `friend class` will forward declare `WindowBase` if need be.

Reply:
Fixed.